Prox/Prox/AppDelegate.swift

```diff
@@ -149,26 +141,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func applicationWillTerminate(_ application: UIApplication) {
         // Called when the application is about to terminate. Save data if appropriate. See also applicationDidEnterBackground:.
         placeCarouselViewController?.locationMonitor.cancelTimeAtLocationTimer()
-        eventsNotificationsManager.persistNotificationCache()
         AppState.exiting()
     }
 
     func application(_ application: UIApplication, performFetchWithCompletionHandler completionHandler: @escaping (UIBackgroundFetchResult) -> Void) {
```

Comment:
If we're going this far, this entire function will never get called with `application.setMinimumBackgroundFetchInterval(AppConstants.backgroundFetchInterval)` removed so you may as well remove the entire thing